xarray/tests/test_backends.py

```diff
@@ -2572,8 +2582,14 @@ def test_open_mfdataset_manyfiles(
             chunks=chunks,
         ) as actual:
 
-            # check that using open_mfdataset returns dask arrays for variables
-            assert isinstance(actual["foo"].data, dask_array_type)
+            try:
+                # check that using open_mfdataset returns dask arrays for variables
+                assert isinstance(actual["foo"].data, dask_array_type)
+            except AssertionError:
+                # A numpy array is returned instead of a dask array
+                # when reading with Zarr and chunks is None
+                if readengine == "zarr" and chunks is None:
+                    assert isinstance(actual["foo"].data, np.ndarray)
```

Comment:
Will need to figure out here why a dask array isn't returned when using `xr.open_mfdataset` with `chunks=None`.

Reply:
Resolved with commit 02f86ca51a5b4182576ac1a66b73fddd011b4d57. Specifically, using `xr.open_mfdataset` with `chunks=None` and `engine="zarr"` should return dask arrays now instead of numpy arrays.